src/main/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReader.java

```diff
@@ -31,39 +41,73 @@
  * {@link org.springframework.batch.item.data.MongoItemReader} is using
  * pagination and it is slow with large collections
  */
-public class NonAnnotatedVariantsMongoReader extends MongoDbCursorItemReader {
+public class NonAnnotatedVariantsMongoReader implements ItemStreamReader<VariantWrapper>, InitializingBean {
```

Comment:
Looks to me a lot of convolution just to convert a DBObject into a VariantWrapper, I can see why we are doing it but I'm not sure if this is the best way. Anyway, I think that `AnnotationProcessor` and the test are no longer necessary then...